src/fides/api/main.py

```diff
@@ -311,14 +311,23 @@ def read_other_paths(request: Request) -> Response:
     if not ui_file:
         ui_file = get_path_to_admin_ui_file(path)
 
-    # If any of those worked, serve the file.
     if ui_file and ui_file.is_file():
-        logger.debug(
-            "catchall request path '{}' matched static admin UI file: {}",
-            path,
-            ui_file,
-        )
-        return FileResponse(ui_file)
+        # We've found a file, but first safety check that this file is really only from our UI directory
+        ui_directory = get_path_to_admin_ui_file("")
```

Comment:
Nit: it's a little cleaner to set the default arg in the `get_path_to_admin_ui_file` function definition as `path: str = ""` then update the docstring that the default functionality is to return the dir path — right now this is effectively "hidden" functionality.

Reply:
good idea!